scene/theme/default_theme.cpp

```diff
@@ -931,8 +931,6 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 	theme->set_stylebox("separator", "VSeparator", separator_vertical);
 
 	theme->set_icon("close", "Icons", icons["close"]);
-	theme->set_font("normal", "Fonts", Ref<Font>());
-	theme->set_font("large", "Fonts", Ref<Font>());
```

Comment:
Note: These two are removed because

* They were used to store default theme fonts when Godot was open sourced. But fonts were changed to a dedicated theme item later, so these entries were abandoned.
* They add unnecessary strings to translate. Not worth adding exceptions for.